### ISSUE TITLE
Max. byte count for FC 16 in Modbus server

### ISSUE THREAD

**Issue by SKittan** [alttch/rmodbus#14]
Why is the max. byte count for FC 16 requests in the Modbus server limited to < 242?

**../server/mod.rs:**
```rust
            MODBUS_SET_COILS_BULK | MODBUS_SET_HOLDINGS_BULK => {
                // funcs 15 & 16
                // write multiple coils / registers
                let bytes = self.buf[self.frame_start + 6];
                if !check_frame_crc!(7 + bytes) {
                    return Err(ErrorKind::FrameCRCError);
                }
                if !broadcast {
                    self.response_required = true;
                }
                if bytes > 242 {  // <== THIS LINE
                    self.error = MODBUS_ERROR_ILLEGAL_DATA_VALUE;
                    return Ok(());
                }
```

If you look into the current Modbus application protocol (V1.1b3), there is denoted a max. quantity of registers for FC 16 of 0x007b. Because the byte count is twice of the register count, the max. allowed byte value should be 246. Or am I missing something?

**Comment by divi255** [comment]
Hi. You are correct, bulk set/get registers is limited to 123 (mathematically even to 124 but most of sources perfer 123). I gonna correct this to set 246 bytes there

**Comment by divi255** [comment]
fixed in 0.7.4
